waspc/packages/deploy/src/providers/fly/DeploymentInstructions.ts

```diff
@@ -0,0 +1,41 @@
+import { CommonCmdOptions } from "./CommonCmdOptions.js";
+import { TomlFilePaths } from "./tomlFile.js";
+
+export type DeploymentInstructions<CmdOptions extends CommonCmdOptions> =
```

Comment:
I first suggested DeploymenInstructions as a name, but then I said oh maybe not because it seems these contain output, not input? I mean they have clientUrl and serverUrl?

Just to check, what are these at the end, is it all input? If so, then DeploymentInstructinos is great name, but if not, then it is confusing.